Example/AppDelegate+CarPlay.swift

```diff
@@ -472,7 +472,27 @@ extension AppDelegate: CPListTemplateDelegate {
 // MARK: - CarPlaySceneDelegate methods
 
 @available(iOS 13.0, *)
-class CarPlaySceneDelegate: NSObject, CPTemplateApplicationSceneDelegate {
+class CarPlaySceneDelegate: NSObject, CPTemplateApplicationSceneDelegate, CPTemplateApplicationDashboardSceneDelegate {
+    
+    @available(iOS 13.4, *)
+    func templateApplicationDashboardScene(_ templateApplicationDashboardScene: CPTemplateApplicationDashboardScene,
+                                           didConnect dashboardController: CPDashboardController,
+                                           to window: UIWindow) {
+        guard let carImage = UIImage(systemName: "car"),
+              let appDelegate = UIApplication.shared.delegate as? AppDelegate else { return }
+        
+        let button = CPDashboardButton(titleVariants: ["Example-CarPlay"], subtitleVariants: ["Mapbox"], image: carImage)
+        dashboardController.shortcutButtons.append(button)
+        
+        let carPlayMapViewController = CarPlayMapViewController(styles: appDelegate.carPlayManager.styles)
```

Comment:
Would it be possible to implement similar method with default implementation on the Navigation SDK side? It'd be great to cover three default states:
- free-drive mode that is using `CarPlayMapViewController`
- routes preview using `CarPlayMapViewController`
- active-navigation using `CarPlayNavigationViewController`

It'd be also great to have the ability to fully override default implementation provided by the Navigation SDK.

I believe that it's also possible to provide guidance instructions for dashboard as well, but it needs verification.